pandas/tests/io/test_sql.py

```diff
@@ -2849,13 +2849,14 @@ def setup_driver(cls):
     def test_keyword_deprecation(self):
         # GH 54397
         msg = (
-            "tarting with pandas version 3.0 all arguments of to_sql except for the "
-            "argument 'name' will be keyword-only."
+            "Starting with pandas version 3.0 all arguments of to_sql except for the  "
+            "arguments 'name' and 'con' will be keyword-only."
         )
         df = DataFrame([{"A": 1, "B": 2, "C": 3}, {"A": 1, "B": 2, "C": 3}])
+        df.to_sql("example", self.conn)
```

Comment:
```suggestion
```

Don't think this is needed?

Reply:
I’d like to keep it to keep a test that this does not raise close by

Reply:
Ah okay. I think you'll need a `if_exists="replace"` below then.